core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/SubmissionImpl.java

```diff
@@ -25,91 +25,87 @@
 
 import java.io.Serializable;
 import java.util.Date;
-import java.util.HashSet;
+import java.util.Map;
 import java.util.Set;
 
-import javax.persistence.CascadeType;
-import javax.persistence.OneToOne;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.MapKeyClass;
+import javax.persistence.MapKeyJoinColumn;
+import javax.persistence.Table;
+import javax.persistence.Transient;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.w3c.dom.Document;
-
-import com.eaglegenomics.simlims.core.SecurityProfile;
-import com.eaglegenomics.simlims.core.User;
 
+import uk.ac.bbsrc.tgac.miso.core.data.Dilution;
+import uk.ac.bbsrc.tgac.miso.core.data.Experiment;
+import uk.ac.bbsrc.tgac.miso.core.data.Sample;
+import uk.ac.bbsrc.tgac.miso.core.data.SequencerPoolPartition;
+import uk.ac.bbsrc.tgac.miso.core.data.Study;
 import uk.ac.bbsrc.tgac.miso.core.data.Submission;
-import uk.ac.bbsrc.tgac.miso.core.data.Submittable;
 import uk.ac.bbsrc.tgac.miso.core.data.type.SubmissionActionType;
-import uk.ac.bbsrc.tgac.miso.core.exception.SubmissionException;
-import uk.ac.bbsrc.tgac.miso.core.manager.SubmissionManager;
-import uk.ac.bbsrc.tgac.miso.core.security.SecurableByProfile;
 
 /**
  * Concrete implementation of a Submission that
  * 
  * @author Rob Davey
  * @since 0.0.2
  */
-public class SubmissionImpl implements Submission<Submittable, Document, Document>, Serializable {
+@Entity
+@Table(name = "Submission")
+public class SubmissionImpl implements Submission, Serializable {
   protected static final Logger log = LoggerFactory.getLogger(SubmissionImpl.class);
 
   public static final Long UNSAVED_ID = 0L;
 
-  private long submissionId = UNSAVED_ID;
-  private String name;
-  private String alias;
   private String accession;
-  private Date creationDate;
-  private Date submissionDate;
-  private boolean verified;
+  private String alias;
   private boolean completed;
+  private Date creationDate;
   private String description;
-  private String title;
+  @ManyToMany(targetEntity = PartitionImpl.class)
```

Comment:
The value of the map should be the target entity. The key (dilution) is `@MapClass`.